docker-images/java-base/scripts/launch_java.sh

```diff
@@ -16,6 +32,6 @@ export MALLOC_ARENA_MAX=2
 JAVA_OPTS="${JAVA_OPTS} -Dvertx.cacheDirBase=/tmp -Djava.security.egd=file:/dev/./urandom"
 
 # Enable GC logging for memory tracking
-JAVA_OPTS="${JAVA_OPTS} ${STRIMZI_GC_LOG_OPTS}"
+JAVA_OPTS="${JAVA_OPTS} $(get_gc_opts)"
```

Comment:
Shouldn't we only do this when `STRIMZI_KAFKA_GC_LOG_ENABLED == true`?

Reply:
I think it returns empty string when `STRIMZI_GC_LOG_ENABLED` is `false`, or?

What I wonder about is whether we really need two environment variables: `STRIMZI_KAFKA_GC_LOG_ENABLED` and `STRIMZI_GC_LOG_ENABLED`. But that is not really issue of this PR.

Reply:
Oh yeah!